docs/source/development/traffic_ops_api/v12/cachegroup_fallbacks.rst

```diff
@@ -0,0 +1,266 @@
+  | ``>name``                         | string | Fallback cache group name                                                |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>order``                        | int    | Ordering list in the list of backups                                     |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+
+  **Request Example** ::
+
+    [
+       {
+          "name": "GROUP2", 
+          "order": 10
+       }
+    ]
+
+  **Response Properties**
+
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | Parameter                         | Type   | Description                                                              |
+  +===================================+========+==========================================================================+
+  |                                   | array  | parameters array                                                         |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>name``                         | string | Fallback cache group name                                                |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>order``                        | int    | Ordering list in the list of backups                                     |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``alerts``                        | array  | A collection of alert messages.                                          |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>level``                        | string | Success, info, warning or error.                                         |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+  | ``>text``                         | string | Alert message.                                                           |
+  +-----------------------------------+--------+--------------------------------------------------------------------------+
+
+
+  **Response Example** ::
+
+    {
+       "alerts": [
+          {
+             "level":"success",
+             "text":"Backup configuration for 1 successful."
+          }
+       ],
+       "response": [
+          {
+             "order":10,
+             "name":"GROUP2"
+          }
+       ]
+    }
+   
+|
+
+**PUT /api/1.2/cachegroups/:id/fallbacks**
+
+  Updates an already existing fallback list for the cache group (Existing fallback list will be deleted). 
```

Comment:
why don't you delete this route and just use POST instead where POST always creates fallbacks and removes the current ones if there are any.

Reply:
May be i should reword that sentence like the following:

Updates / replaces an already existing fallback list for the cache group.

Reply:
Ok. Makes sense. I agree that we can do away with PUT. Hope every one is fine with this.